pkg/controller/certificates/issuing/temporary.go

```diff
@@ -68,8 +68,9 @@ func (c *controller) ensureTemporaryCertificate(ctx context.Context, crt *cmapi.
 		return false, err
 	}
 	secretData := internal.SecretData{
-		Certificate: certData,
-		PrivateKey:  pkData,
+		Certificate:     certData,
+		PrivateKey:      pkData,
+		CertificateName: crt.Name,
```

Comment:
So when a certificate is issued, this is how the certificate name annotation value gets populated.
What about IssuerName, IssuerKind, IssuerGroup? Don't those also get added as annotations to this temporary secret?

Reply:
I did not add those annotations for a temporary secret, since the temporary secret actually does not use the configured issuer (I try to prevent the user from being confused).
Not sure if there is a technical reason to (not) set those values here.